aiida/orm/nodes/process/process.py

```diff
@@ -134,6 +144,23 @@ def process_class(self) -> Type['Process']:
 
         return process_class
 
+    @property  # type: ignore
+    @field_method()
+    def process_type(self) -> Optional[str]:
```

Comment:
I think `process_type` should be removed from the base `Node` class, since obviously it is just a process specific field and an implementation detail that it is also contained in the database table for all nodes

Reply:
(same for `computer`, that is only relevant to code amd process nodes)

Reply:
No, computer should stay for all nodes I think, some other nodes use it (specifically: RemoteData). Default = None.

Reply:
It should definitely stay on the `Node` level.

Reply:
reverted this change as no longer necessary